web/modules/custom/dpl_login/dpl_login.info.yml

```diff
@@ -7,3 +7,4 @@ core_version_requirement: ^9
 
 dependencies:
   - openid_connect:openid_connect
+  - dpl_react_apps:dpl_react_apps
```

Comment:
Consider if this dependency can be handled on other ways. 

I am not too keen on it as it creates a dependency on a module which (should) also depend on `dpl_login`. I get what you want to do but I fear a circular relationship where I personally see DPL Login as a dependency on the React Apps and not the other way around.

One way to do this would be to let DPL React Apps depend on DPL Login and generate the urls in its own `dpl_react_apps_dpl_react_apps_data()` implementation.

Reply:
You're right I'll move it around